src/mui/input/FileInput.js

```diff
@@ -39,8 +43,6 @@ export class FileInput extends Component {
     };
 
     static defaultProps = {
-        addLabel: true,
```

Comment:
Shouldn't it have a default `addLabel: false` ?

Reply:
`addLabel` is no longer used in Input components, that's why I added a `<Labeled>` component in the `render()` method.